cluster/ci/config/prow/config.yaml

```diff
@@ -76,7 +76,6 @@ tide:
     openshift/jenkins-openshift-login-plugin: rebase
     openshift/ansible-service-broker: rebase
     openshift/template-service-broker-operator: rebase
-    openshift/cluster-api: rebase
```

Comment:
I actually like the rebase strategy. Though, not a blocker here.

Reply:
please, see and chime in https://docs.google.com/document/d/1yUHXJi7LEQuJSdd5cBUJVY-D74_bmg_WC2xdDwayP6I/edit#